app/controllers/solidus_pay_tomorrow/create_order_controller.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module SolidusPayTomorrow
+  class CreateOrderController < BaseController
+    def create
+      order = Spree::Order.find_by(number: permitted_params[:order_number])
+      payment_method = SolidusPayTomorrow::PaymentMethod.find(permitted_params[:payment_method])
+      @result = SolidusPayTomorrow::Client::CreateOrderService.call(order: order, payment_method: payment_method)
+      @successfully_created = @result.key?('url') && @result.key?('token')
+      if @successfully_created
+        render 'spree/checkout/payment/_checkout', status: :created
+      end
+    rescue StandardError => e
+      @error_message = e.message
+      render 'spree/checkout/payment/_checkout', status: :unprocessable_entity
```

Comment:
It should be a JSON

Reply:
Changed in https://github.com/nebulab/solidus_pay_tomorrow/pull/9/commits/bd16050623ac9409f748fc49c53b2056f9636922